### ISSUE TITLE
Forbid properties with parameters

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#698]
# Rule request

## Thesis

Code like this:

```python
class Test(object):
    @property
    def b(self, arg):
        return 1
```

Should be forbidden. Since this class definition is incorrect.
`.b` won't work at all.

## Reasoning

This is OOP rule that checks that properties are defined correctly.

**Comment by sobolevn** [comment]
We should use `@property` and `abc.abstractbaseproperty` as decorators.

**Comment by sobolevn** [comment]
`mypy` also catches this:

```python
class Test:
    @property
    def test(self, abc):
        return 1

Test().test
```

Returns:

```
» mypy ex.py
ex.py:3: error: Too many arguments
```

**Comment by sobolevn** [comment]
We do not assume or check types.
